docs/UpgradeGuide.md

```diff
@@ -90,14 +85,14 @@ Datadog.configure do |c|
   c.agent.hostname = '127.0.0.1'
   c.agent.port = 8126
   c.diagnostics.debug = true
```

Comment:
I disagree: although I can understand why it would be appealing to keep ordering the same logically, visual appeal of the outcome is compromised if sensible groups/alphabetical order isn't maintained when converted.

Reply:
I think we're in alignment. My suggestion was not to break the new groups, but to reorder things in the new group in the same order as they appeared in the 0.x config:

```ruby
  # Global settings
  c.diagnostics.debug = true
  c.service = 'billing-api'
  c.runtime_metrics.enabled = true
  c.agent.hostname = '127.0.0.1'
  c.agent.port = 8126
```